test/cypress/integration/group-paying.spec.js

```diff
@@ -325,6 +325,11 @@ describe('Group Payments', () => {
 
     cy.log('user1 receives a notification for a thank you note')
     cy.giSwitchUser(`user1-${userId}`)
+    // The following 'wait' addresses an issue whereby the 'thank you'
+    // notification isn't yet rendered.
+    /* cy.window().its('sbp').then(sbp => {
+      return sbp('chelonia/contract/wait', sbp('state/vuex/state').currentGroupId)
+    }) */
```

Comment:
A comment is added here saying an issue is being addressed, and then the code is commented out - achieving nothing. Perhaps this whole section should be removed?

Reply:
This was an attempt at addressing an issue that I only ever saw in GHA and is therefore difficult to debug. I suspected that this would fix it, but then I saw the issue still happening. Ideally, we want to fix the underlying issue, but either way it looks like this should be safe to remove.